code/game/objects/items/toys.dm

```diff
@@ -449,6 +449,12 @@
 	..()
 	name = "[colourName] crayon" //Makes crayons identifiable in things like grinders
 	drawtype = pick(pick(graffiti), pick(letters), "rune[rand(1,6)]")
+	if(config != null)
+		if(config.mutant_races == 1)
+			graffiti += "antilizard"

```

Comment:
Since these grafitti are added in both New() and initialize(), change both of them from ```+=``` to ```|=``` to prevent duplicates.